### PR TITLE
Format numbers with commas and two decimal places

### PR DESCRIPTION
**Pull request by phonz-dev**
This commit implements number formatting for displayed values on the website. Numbers are now formatted with commas as thousands separators and rounded off to two decimal places for improved readability.

Before:
before-table
before-no-comma

After:
redeem-table
number-commas

This PR closes issue #59

### CODE REVIEW

**Review by Mercurial — APPROVED** [review]

**Comment by Mercurial** [comment]
this looks good waiting on your approval to merge @esot321c

**Comment by esot321c** [comment]
This way uses their locale, rather than force `en-us`, and drops digits if they aren't needed (like 1.00 would be 1)

`string.toLocaleString(undefined, { maximumFractionDigits: 2, })`

**Pull request merged**
